homeassistant/components/bluetooth/manager.py

```diff
@@ -344,6 +346,8 @@ def _async_check_unavailable(self, now: datetime) -> None:
                     # by the lack of advertisements
                     if advertising_interval := intervals.get(address):
```

Comment:
```suggestion
                    if advertising_interval := (intervals.get(address) or self._fallback_intervals.get(address)):
```

Reply:
Little longer but at least we avoid writing `advertising_interval += TRACKER_BUFFERING_WOBBLE_SECONDS` twice

Reply:
Yeah I went backwards and forwards on that line a few times 😂